robot.py

```diff
@@ -35,26 +36,33 @@ def follow_line(self, millies):
         while self.tank.distance() < millies:
             subtract = self.left_cs.reflection() - self.right_cs.reflection() 
             multiply = subtract * SHARPNESS_COLOR
-            self.tank.drive(SPEED, multiply) 
+            self.tank.drive(speed, multiply) 
+        self.brake()
+
+    def brake(self):
+        ''' brakes the robot
```

Comment:
I would have removed obvious comments, e.g., when it is not adding much more to the name of a function.